Tests/scripts/runTests.py

```diff
@@ -103,6 +106,7 @@ def RunTestCase(asn1, acn, behavior, expErrMsg):
             PrintFailed("coverage failed. (less than 100%)")
             sys.exit(1)
     else:
+        prevDir = os.getcwd()
```

Comment:
On mac os, this changed from _Tests/tmp_ to _Tests/scripts_, and line 168 went up instead of back, which would cause the next test case to fail since it could not find **Asn1f2.exe**.